docs/cli-manual.md

```diff
@@ -1,39 +1,86 @@
-# The TiDB Kubernete Contorl(tkc) User Manual
+# The TiDB Kubernete Contorl(tkctl) User Manual
 
-> **Disclaimer**: The tkc CLI tool is currently **Alpha**. The design and sub-commands may change in the future, use at your own risk.
+> **Disclaimer**: The tkctl CLI tool is currently **Alpha**. The design and sub-commands may change in the future, use at your own risk.
 
-The TiDB Kubernetes Control(tkc) is a command line utility for TiDB operators to operate and diagnose their TiDB clusters in kubernetes.
+The TiDB Kubernetes Control(tkctl) is a command line utility for TiDB operators to operate and diagnose their TiDB clusters in kubernetes.
 
 - [Installation](#installation)
+    - [Bash Completion](#bash-completion)
     - [Kubernetes Configuration](#kubernetes-configuration)
 - [Commands](#commands)
-    - [tkc list](#tkc-list)
-    - [tkc use](#tkc-use)
-    - [tkc info](#tkc-info)
-    - [tkc get](#tkc-get)
-    - [tkc debug](#tkc-debug)
-    - [tkc ctop](#tkc-ctop)
-    - [tkc help](#tkc-help)
-    - [tkc options](#tkc-options)
-- [Appendix](#appendix)
-    - [Use tkc as kubectl plugin](#use-tkc-as-a-kubectl-plugin)
-    - [Auto Completion](#auto-completion)
+    - [tkctl version](#tkctl-version)
+    - [tkctl list](#tkctl-list)
+    - [tkctl use](#tkctl-use)
+    - [tkctl info](#tkctl-info)
+    - [tkctl get](#tkctl-get)
+    - [tkctl debug](#tkctl-debug)
+    - [tkctl ctop](#tkctl-ctop)
+    - [tkctl help](#tkctl-help)
+    - [tkctl options](#tkctl-options)
 
 # Installation
 
+You can download the pre-built binary or build `tkctl` from source:
+
+### Download the Pre-built Binary
+
+- [MacOS](http://download.pingcap.org/tkctl-darwin-amd64-latest.tgz)
+- [Linux](http://download.pingcap.org/tkctl-linux-amd64-latest.tgz)
+- [Windows](http://download.pingcap.org/tkctl-windows-amd64-latest.tgz)
+
+### Build from Source
+
+```shell
+$ git clone https://github.com/pingcap/tidb-operator.git
+$ GOOS=${YOUR_GOOS} make cli
+$ mv tkctl /usr/local/bin/tkctl
+```
+
+## Bash Completion
+
+BASH
+```shell
+# setup autocomplete in bash into the current shell, bash-completion package should be installed first.
+source <(tkctl completion bash) 
+
+# add autocomplete permanently to your bash shell.
+echo "source <(tkctl completion bash)" >> ~/.bashrc 
+```
+
+ZSH
+```shell
+# setup autocomplete in zsh into the current shell
+source <(tkctl completion zsh)
+
+# add autocomplete permanently to your zsh shell
+echo "if [ $commands[tkctl] ]; then source <(tkctl completion zsh); fi" >> ~/.zshrc 
```

Comment:
You can use the command `echo "if  hash tkctl 2>/dev/null; then source <(tkctl completion zsh); fi" `  for both bash and zsh